core/src/Revolution/Processors/System/Derivatives/GetList.php

```diff
@@ -35,10 +37,44 @@ public function initialize()
     {
         $this->setDefaultProperties([
             'class' => '',
+            'restrict_actions' => '',
         ]);
         return true;
     }
 
+    /**
+     * @param array $descendents
+     * @return array
+     */
+    protected function checkRestrictions(array $descendents): array
+    {
+        $skip = explode(',', $this->getProperty('skip'));
+        $actions = explode(',', $this->getProperty('restrict_actions'));
+
+        $map = [
+            'MODX\Revolution\modWebLink' => 'weblink',
+            'MODX\Revolution\modSymLink' => 'symlink',
+            'MODX\Revolution\modStaticResource' => 'static_resource',
+        ];
+
+        foreach ($actions as $action) {
+            if (!in_array($action, self::RESTRICT_ACTIONS)) {
+                continue;
+            }
+
+            foreach ($descendents as $descendent) {
+                if ($descendent === 'MODX\Revolution\modDocument') {
```

Comment:
It's better to use `::class` to get the FQC instead of using a string.

Reply:
Thanks, done!